### PR TITLE
Cofense mapping

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Cofense mapping

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py [hunk 1]
```diff
@@ -157,8 +157,27 @@ def process_item(cls, threat: dict) -> List[dict]:
...
-                results.append({"value": value, "type": indicator, "rawJSON": block})
+                results.append({
+                    "value": value,
+                    "type": indicator,
+                    "rawJSON": block,
+                    "fields": {
+                        "name": threat_id,
+                        "malwarefamily": malware_family.get("familyName"),
+                        "description": malware_family.get("description"),
+                        "feedoriginalseverity": block.get("impact"),
```
Comment (avidan-H) [inline Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py]: Please check what happens if these values are `None`. If it's problematic then you'll need to set the default argument to `get()` to be of the appropriate type that given indicator field expects.

File: Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py [hunk 2]
```diff
@@ -157,8 +157,27 @@ def process_item(cls, threat: dict) -> List[dict]:
...
+                        "feedthreattypes": {
+                            "threatcategoryconfidence": block.get("confidence"),
+                            "threatcategory": block.get("role")
+                        },
+                        "geocountry": ip_detail.get("countryIsoCode"),
```
Comment (avidan-H) [inline Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py]: The same goes for here as well.

File: Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py (hunk 1 above)
Comment (guyfreund) [reply Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py]: Checked, no exception is raised. Indicators with missing fields are being fetched as usual.

File: Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py (hunk 2 above)
Comment (guyfreund) [reply Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py]: Checked, no exception is raised. Indicators with missing fields are being fetched as usual.

**Pull request merged**